src/components/molecules/definitionitem/DefinitionItem.scss

```diff
@@ -0,0 +1,8 @@
+.results dt {
```

Comment:
these can be nested

Reply:
so this is a mock component - was just a placeholder